### DISCUSSION TITLE
[Week 27] Request for Contributions

### DISCUSSION THREAD

**Discussion by bdougie** [open-sauced/open-sauced#750] [Newsletter]
After a week off, I am continuing with the Request for Contributions. This list was sourced from [ups-for-grabs](https://up-for-grabs.net/).

# What is this? 👀
There are several projects looking for contributions from experience and first-time contributors. Below is a list of those projects.

### TypeScript/JavaScript
- [vue](https://github.com/vuejs/vue/contribute) - 🖖 Vue.js is a progressive, incrementally-adoptable JavaScript framework for building UI on the web.
- [vue-carousel](https://github.com/SSENSE/vue-carousel/contribute) - A flexible, responsive, touch-friendly carousel for Vue.js
- [remirror](https://github.com/remirror/remirror/contribute) - A universal react text editor built with prosemirror

### GitHub Action Workflow improvement
- [pandas](https://github.com/pandas-dev/pandas/issues/32301) - _(Looking for help with GitHub Actions_) - Flexible and powerful data analysis / manipulation library for Python, providing labeled data structures similar to R data.frame objects, statistical functions, and much more

### Open Sauced

- Images flicker when page is reload the homepage. #612
- Pagination: Unhandled Rejection (TypeError): Cannot read property 'gitHub' of null #535
- Feature: Create a Changelog #668

## :mag: Needs Triage Help
the following projects could use help triaging old issues by marking them as valid or invalid. Considering stopping by the below projects and offering assistance.  

@open-sauced/triage 

- [moment-timezone](https://github.com/moment/moment-timezone) - Timezone support for moment.js
- [brackets](https://github.com/adobe/brackets) - An open source code editor for the web, written in JavaScript, HTML and CSS.

**Comment by t8** [comment]
<h1><a href="https://github.com/mandarineorg/mandarinets">Mandarine.TS</a></h1>

Mandarine.TS is a framework that allows you to create applications with Deno.

I'm a huge fan of the way the framework is built out, and I would love to see some contributors that have time to help Andres out with testing, bug fixes, and new features.

More specifically, I think he could use some assistance working towards getting Mandarine production-ready. See this issue for more information: https://github.com/mandarineorg/mandarinets/issues/57

Thanks!
